gobblin-service/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -223,6 +223,8 @@ public static GobblinServiceManager createTestGobblinServiceManager(Properties s
 
     DagManager spiedDagManager = spy(gobblinServiceManager.getDagManager());
     doNothing().when(spiedDagManager).setActive(anyBoolean());
+    // WARNING: this `spiedDagManager` WILL NOT BE the one used by the `Orchestrator`: its DM has apparently already been
+    // provided to the `Orchestrator` ctor, prior to this replacement here of `GobblinServiceManager.dagManager`
```

Comment:
documenting for posterity, since unfortunately, this was quite troublesome to debug